Automattic/wp-calypso
# Emails: Cleanup unused GSuite new users code

#### Proposed Changes

#65088, #65180, and #67420 remove a bunch of legacy code related to GSuite usages. This PR is a follow-up that removes the unused collection of GSuite new user creation and validation helper functions. I'm refactoring the only usage, which happens to be a duplication of a type that exists in the `@automattic/shopping-cart` package.

#### Testing Instructions

Verify the removed code is not in use. Matticbot should report no bundle size changes.
